src/ResourceManager/Properties/Resources.Designer.cs

```diff
@@ -65,8 +65,8 @@ public class Resources {
         ///The data is anonymous and does not include commandline argument values.
         ///The data is collected by Microsoft.
         ///
-        ///Use the Disable-AzureRmDataCollection cmdlet to turn the feature Off. The cmdlet can be found in the AzureRM.Profile module.  To disable data collection: PS &gt; Disable-AzureRmDataCollection.
-        ///Use the Enable-AzureRmDataCollection cmdlet to turn the feature On. The cmdlet can be found in the AzureRM.Profile module.  To enable  [rest of string was truncated]&quot;;.
+        ///Use the Disable-AzureRmDataCollection cmdlet to turn the feature Off. The cmdlet can be found in the AzureRM.Accounts module.  To disable data collection: PS &gt; Disable-AzureRmDataCollection.
```

Comment:
Same comment - I thought these would be the same as below

Reply:
Fixed.